jujugui/static/gui/src/app/app.js

```diff
@@ -1230,7 +1230,8 @@ YUI.add('juju-gui', function(Y) {
       const env = this.env;
       const db = this.db;
       const connected = this.env.get('connected');
-      const modelName = connected ? db.environment.get('name') : 'mymodel';
+      const environmentName = env.get('environmentName')
+      const modelName = environmentName ? environmentName : 'mymodel';
```

Comment:
Why not just `const modelName = env.get('environmentName') || 'mymodel';`?
Let's not reintroduce the "environment" to mean "model" please.